libs/analytics/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { debounce } from '@cowprotocol/common-utils'
+
+import { CowAnalytics } from './CowAnalytics'
+import { Category } from './types'
+
+export const createDebouncedTradeAmountAnalytics = (cowAnalytics: CowAnalytics) => {
```

Comment:
It looks very specific to `apps/cowswap-frontend/src/modules/trade` and is supposed to be there

Reply:
Should be addressed